Containerfile.dependencies

```
@@ -0,0 +1,9 @@
+FROM python:3.12
```

```suggestion
FROM python:3.13
```
Let's update to 3.13 while we're making changes?